tests/slurm-tests/gpt_oss_python_aime25/run_test.py

```diff
@@ -13,6 +13,12 @@
 # limitations under the License.
 
 import argparse
+import sys
+from pathlib import Path
+
+# Add parent directory to path to import utils
+sys.path.insert(0, str(Path(__file__).parents[1]))
```

Comment:
For consideration: If there are common utilities, then perhaps we can move them under `nemo_skills` itself, and avoid the ugly `sys.path`.